### ISSUE TITLE
How do I reference the 'numpy' library

### ISSUE THREAD

**Issue by zhaocheng-hub** [numpy/numpy#18595]
Hello, I have used some methods of 'numpy' library in my academic paper. How should I quote them in my paper?

**Comment by mattip** [comment]
hmm, maybe it should be mentioned at https://numpy.org/press-kit/ ? [cite-bay](http://citebay.com/how-to-cite/numpy/) suggests the 2006 *A guide to NumPy* book, but I think it would be better to cite the [2020 Nature Methods](https://www.nature.com/articles/s41586-020-2649-2#citeas) paper.

**Comment by rkern** [comment]
The _Nature Methods_ paper is the preferred citation these days.

Information on citing NumPy and SciPy is here: https://www.scipy.org/citing.html

There probably should be something on numpy.org as well.

**Comment by melissawm** [comment]
Yes, it is here: https://numpy.org/learn/ under [Citing NumPy](https://numpy.org/citing-numpy/) - maybe this should be more obvious/findable?

**Comment by seberg** [comment]
Maybe it makes more sense in the "About Us" section?  Or as a quick access in the bottom?

**Comment by mattip** [comment]
> Or as a quick access in the footer?

+1

**Comment by mattip** [comment]
we really need unified search across the numpy.org website, or at least I should have tried `site:numpy.org citing` which points out the "citing" page
